test/navigation/NavigatorSpec.scala

```diff
@@ -1260,6 +1367,30 @@ class NavigatorSpec extends SpecBase with BeforeAndAfterEach {
               routes.CategorisationResultController.onPageLoad(testRecordId, Category2Scenario)
           }
 
+          ".NIPHL assesments" - {
+
+            "to category 2 result page when categorisation result is so" in {
+              val categoryInfoWithNiphlAssessments = CategorisationInfo(
```

Comment:
This isn't actually testing anything different from the current Category2Scenario test, because the mock is driving the behaviour rather than the UA?

Reply:
Removed